cluster/saltbase/salt/etcd/initd

```diff
@@ -40,6 +40,10 @@ DAEMON_USER=etcd
 #
 do_start()
 {
+        # Raise the file descriptor limit - we expect to open a lot of sockets
+        # (etcd stores file per object).
```

Comment:
Actually I still this error, but much less often:

E0327 14:54:52.018144    4512 poller.go:72] failed to list: Get http://127.0.0.1:4001/v2/keys/registry/minions?quorum=false&recursive=true&sorted=true: dial tcp 127.0.0.1:4001: too many open files

@xiang90 any suggestion?